packages/rum/test/benchmarks/analyzer.js

```diff
@@ -28,19 +28,48 @@ const stats = require('stats-lite')
 const { readFileSync } = require('fs')
 const path = require('path')
 const zlib = require('zlib')
+const webpack = require('webpack')
+const {
+  getWebpackConfig,
+  BUNDLE_TYPES
+} = require('../../../../dev-utils/build')
 const { runs, noOfImages } = require('./config')
 
 const dist = path.join(__dirname, '../../dist')
 
-function getMinifiedApmBundle() {
-  return readFileSync(
-    path.join(dist, 'bundles/elastic-apm-rum.umd.min.js'),
-    'utf-8'
-  )
+function customApmBuild(filename) {
+  /**
+   * Match it with the default webpack prod build of elasticApm
+   * expect the function names are not mangled
+   */
+  const config = {
+    entry: path.join(__dirname, '../../src/index.js'),
+    output: {
+      filename,
+      path: path.join(dist, 'bundles'),
+      library: '[name]',
+      libraryTarget: 'umd'
+    },
+    ...getWebpackConfig(BUNDLE_TYPES.BROWSER_ESM_PROD)
```

Comment:
What do you think about moving the configs to dev-utils? e.g. having a separate function for getting the benchmark configs. One plus of this is that we can only set `keep_fnames: true ` on this configuration instead of changing the default production config.

Reply:
`build.js` is a common for all the builds. I prefer to keep the configs along with the files that are actually doing the job, we can definitely do the same `keep_fname: true` here also without overrding the default one. it would be the same wherever we keep this config.